src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,284 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<Task> ();
+    private static int numberOfTasks;
+    private static boolean addTask;
+
+    private static String logo = "     ____        _        \n" +
```

Comment:
if it doesnt change, can make it `final`

Reply:
I see! Will make all constants final!